storybook-docs/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@fondue/docs",
+  "type": "module",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "storybook": "storybook dev -p 6010",
+    "build:copyComponentsStorybook": "mkdir -p storybook-static/new/ && cp -r ../packages/components/storybook-static/. storybook-static/new",
+    "build:copyFondueStorybook": "mkdir -p storybook-static/legacy/ && cp -r ../packages/fondue/storybook-static/. storybook-static/legacy",
+    "build:storybook": "storybook build && pnpm run build:copyComponentsStorybook && pnpm run build:copyFondueStorybook"
+  },
+  "dependencies": {
+    "@frontify/fondue": "workspace:^",
+    "@frontify/fondue-components": "workspace:^"
```

Comment:
there shouldn't be any dependency here, this package should be fully isolated from anything related to our packages

Reply:
agree, removed